content/en/docs/components/pipelines/sdk/connect-api.md

```diff
@@ -0,0 +1,118 @@
++++
+title = "Connecting to API"
+description = "How to connect to Kubeflow Pipelines API using the SDK and configure it using environment variables"
+weight = 80
++++
+
+This guide introduces:
+
+* how to connect to your Kubeflow Pipelines API using [the Kubeflow Pipelines SDK](/docs/components/pipelines/sdk/sdk-overview/) in general.
+* how to [configure the SDK client using environment variables](#configure-sdk-client-by-environment-variables).
+
+## Before you begin
+
+* You need a Kubeflow Pipelines deployment using one of the [installation options](/docs/components/pipelines/installation/overview/).
+* [Install the Kubeflow Pipelines SDK](/docs/components/pipelines/sdk/install-sdk/).
+
+## How Kubeflow Pipelines SDK connects to API?
+
+Kubeflow Pipelines REST API is available in the same endpoint as its UI.
+The SDK client can send requests to this endpoint to upload pipelines, create pipeline runs, schedule recurring runs and more.
+
+Usually, Kubeflow distributions secure the endpoint with authentication & authorization, so the exact steps needed to connect to your own Kubeflow Pipelines API is different depending on the Kubeflow distribution you installed, because they may have different authentication requirements. Refer to documentation for [your Kubeflow distribution](/docs/started/installing-kubeflow/):
+
+* [Connecting to Kubeflow Pipelines on Google Cloud using the SDK](/docs/distributions/gke/pipelines/authentication-sdk/)
+* [Kubeflow Pipelines on AWS](https://www.kubeflow.org/docs/distributions/aws/pipeline/#authenticate-kubeflow-pipeline-using-sdk-inside-cluster)
+* [Authentication using OIDC in Azure](/docs/distributions/azure/authentication-oidc/)
+* [Pipelines on IBM Cloud Kubernetes Service (IKS)](/docs/distributions/ibm/pipelines/)
+
+If the endpoint is accessible without authentication / authorization, it's easier to connect. Here are examples
+for [Kubeflow Pipelines standalone](https://www.kubeflow.org/docs/components/pipelines/installation/standalone-deployment/).
+
+### Connect to API from outside your cluster
+
+Kubeflow Pipelines standalone deploys a Kubernetes service named `ml-pipeline-ui` in your Kubernetes cluster without extra authentication.
+
+You can use [kubectl port-forward](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/) to port forward the Kubernetes service locally to your laptop outside of the cluster:
+
+```bash
+# Change the namespace if you deployed Kubeflow Pipelines in a different
+# namespace.
+$ kubectl port-forward svc/ml-pipeline-ui 3000:80 --namespace kubeflow
+```
+
+You can verify port forwarding is working properly by visiting <http://localhost:3000> in your browser. Kubeflow Pipelines UI should show up.
```

Comment:
verify port -> verify that port
<http://localhost:3000> -> [http://localhost:3000](http://localhost:3000)
Kubeflow Pipelines UI should show up. -> If port forwarding is working properly, the Kubeflow Pipelines UI appears.